### PR TITLE
[OBSDEF-8781] Add labels to enable logging sidecar injection for fedsvc

### PR DESCRIPTION
**Pull request by voa808**
[OBSDEF-8781] Add labels to enable logging sidecar injection for fedsvc
Add fedsvc log configMap
Add fedsvc log4j2 conf mapping from k8s configmap

Code changes for logs in file are here: https://eos2git.cec.lab.emc.com/ECS/federation-service/pull/71

## PR checklist
- [ ] make test passed
- [x] make build passed
- [ ] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed
- [ ] Passed - https://asd-ecs-jenkins.isus.emc.com/jenkins/view/ECS-Flex/job/charts-custom-ci/

## Testing
https://asd-ecs-jenkins.isus.emc.com/jenkins/job/charts-custom-ci/193/

_Paste the output of your helm install/vSphere7 deployment testing here_

### CODE REVIEW

File: federation/templates/fedsvc-log-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: objectscale-manager
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/part-of: ecs-flex-manager
+data:
+  fedsvc-log4j2.xml: |
```
Comment (lipak2345) [inline federation/templates/fedsvc-log-configmap.yaml:14]: So is this config used for logging or the file defined in fedsvc PR (https://eos2git.cec.lab.emc.com/ECS/federation-service/pull/71/files) fedsvc/src/conf/fedsvc-log4j2.xml?

Comment (voa808) [reply federation/templates/fedsvc-log-configmap.yaml:14]: @lipak2345 I updated the configmap, changed INFO to Debug level and it reflected the change in the logs 
https://jira.cec.lab.emc.com/browse/OBSDEF-8781?focusedCommentId=6120213&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-6120213

Comment (voa808) [reply federation/templates/fedsvc-log-configmap.yaml:14]: Checked the K8s documentation, we are populating a volume with data stored in the configMap. Reference: https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#populate-a-volume-with-data-stored-in-a-configmap

**Review by SeemaTahaliyani — APPROVED** [review]

**Review by lipak2345 — APPROVED** [review]

**Comment by voa808** [comment]
@eddiepavkovic Could you please add fedsvc to the list so that service pod picks up the federation logs from rsyslog.

**Pull request merged**